### DISCUSSION TITLE
Open pdf file in the current tab

### DISCUSSION THREAD

**Discussion by valuex** [sumatrapdfreader/sumatrapdf#4077] [Ideas]
**Requested Feature**:  Open pdf file in the current tab by DDE or by receiving WM_COPYDATA
**The reason I make this request**  
I would like to preview dozens of pdf file continiously and make some notes for those pdf files.  
I can do continious previewing in Total Commander, but it is a file manger, not good for taking some notes for the corresponding file.
So I'd like to use excel and SumatraPDF together to do continious previewing and taking notes.
Luckily, with SumatraPDFControl I can develop a WinForm application and get the message from Excel by VBA.
But still, I raise this request here that because the application I develped is only for previewing purpose, and I get to open it only for the previewing purpose. while for SumatraPDF, I always keep it open. So I think it would be better to add a small feature in SumatraPDF.

**Comment by kjk** [comment]
This should be an issue. Bug reports and feature requests posted in discussions will be forgotten.

**Reply by valuex** [reply]
Sorry. I posted an request in the issue part. Will close this. Thanks.
